src/main/io/osd.c

```diff
@@ -640,6 +736,25 @@ static bool osdDrawSingleElement(uint8_t item)
             break;
         }
 
+    case OSD_MESSAGES:
+        {
+            const char *message = NULL;
+            if (ARMING_FLAG(ARMED)) {
+                // TODO: Add some messages here
+            } else if (IS_RC_MODE_ACTIVE(BOXARM)) {
```

Comment:
We may be trying to arm via sticks. We need to show the reason for blocked arming regardless of arming switch

Reply:
Hadn't thought of that. I'll add an ìsTryingToArm()` function or something named like that to rc_modes.h and I'll use in osd.c.

Reply:
Upon further inspection, `rc_controls.h` seems like a better to place to add the `isTryingToArm()` function.